specification/web/resource-manager/Microsoft.DomainRegistration/stable/2024-11-01/Domains.json

```diff
@@ -0,0 +1,1616 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-11-01",
+    "title": "Domains API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DomainRegistration/checkDomainAvailability": {
```

Comment:
>/subscriptions/{subscriptionId}/providers/Microsoft.DomainRegistration/checkDomainAvailability [](http://example.com/codeflow?start=5&length=94)

New tenant-level APIs require review by the PAS team: [https://aka.ms/azurerbacofficehours](https://aka.ms/azurerbacofficehours).